pandas/tests/copy_view/test_methods.py

```diff
@@ -721,3 +721,22 @@ def test_squeeze(using_copy_on_write):
         # Without CoW the original will be modified
         assert np.shares_memory(series.values, get_array(df, "a"))
         assert df.loc[0, "a"] == 0
+
+
+def test_items(using_copy_on_write):
+    df = DataFrame({"a": [1, 2, 3], "b": [4, 5, 6], "c": [7, 8, 9]})
+    df_orig = df.copy()
+    for name, vals in df.items():
+        ser = Series(vals)
```

Comment:
Does `vals` need to be rewrapped in `Series`? vals is already a `Series`

Reply:
Thanks updated.